pkgs/applications/blockchains/stellar-core.nix

```diff
@@ -1,29 +1,25 @@
-{ lib, stdenv, fetchgit, autoconf, libtool, automake, pkg-config, git
+{ lib, stdenv, fetchFromGitHub, autoconf, libtool, automake, pkg-config, git
 , bison, flex, postgresql }:
 
-let
+stdenv.mkDerivation rec {
   pname = "stellar-core";
-  version = "0.5.1";
+  version = "17.0.0";
 
-in stdenv.mkDerivation {
-  name = "${pname}-${version}";
-
-  src = fetchgit {
-    url = "https://github.com/stellar/stellar-core.git";
-    rev = "refs/tags/v${version}";
-    sha256 = "0ldw3qr0sajgam38z2w2iym0214ial6iahbzx3b965cw92n8n88z";
+  src = fetchFromGitHub {
+    owner = "stellar";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1bq5j69zb9bhkqmz7mkz5s47p385szjd9pvm3pp3h898qc5m6am8";
```

Comment:
```suggestion
    sha256 = "09vpznc0iw6y8lg0h39yfrjacapvs47wl382vbfg1xrfi1hks30l";
```

Reply:
Get `1d1r5q1pac8bxffaz075980dfi4f8sh8vj8gcc1cz4icvsby1x50` here. Could be an instance of #8567

Reply:
I'm glad it's not just me somehow! Thank you for that link, I will try out some workarounds.